docs/API.md

```diff
@@ -185,6 +186,17 @@ Type: `string`
 Set a custom path for the error overlay to listen to Webpack build messages.
 Useful if you set `devServer.sockPath` to something other than `/sockjs-node`.
 
+### `sockProtocol`
+
+Default: parsed from current url
+
+Type: `enum`
```

Comment:
I think it would be better for use to be explicit about this - what about:

```suggestion
Type: `http:` or `https:`
```

(refernce `sockIntegration`)

Reply:
As a note to that, I think `ws` and `wss` are also technically valid choices if people use strict, native WebSockets (not `SockJS`)

Reply:
Another note I think would be relavent - I'm more inclined to have `http` instead of `http:` here. I think we should either go the full way (`http://`) or just go with the protocol name (`http`).

Reply:
Yup, I wanted to do that, just wasn’t sure. Updated